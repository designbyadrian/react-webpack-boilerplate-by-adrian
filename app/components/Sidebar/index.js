import React from 'react';
import PropTypes from 'prop-types';

import './style.module.sass';

class Sidebar extends React.Component {
  render() {
    return (
      <aside styleName="sidebar">
        <div className="list-group">
          <a href="#" className="list-group-item list-group-item-action flex-column align-items-start">
            <div className="d-flex w-100 justify-content-between">
              <h5 className="mb-1">List group item heading</h5>
              <small>3 days ago</small>
            </div>
            <p className="mb-1">Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.</p>
            <small>Donec id elit non mi porta.</small>
          </a>
          <a href="#" className="list-group-item list-group-item-action flex-column align-items-start">
            <div className="d-flex w-100 justify-content-between">
              <h5 className="mb-1">List group item heading</h5>
              <small className="text-muted">3 days ago</small>
            </div>
            <p className="mb-1">Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.</p>
            <small className="text-muted">Donec id elit non mi porta.</small>
          </a>
          <a href="#" className="list-group-item list-group-item-action flex-column align-items-start">
            <div className="d-flex w-100 justify-content-between">
              <h5 className="mb-1">List group item heading</h5>
              <small className="text-muted">3 days ago</small>
            </div>
            <p className="mb-1">Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.</p>
            <small className="text-muted">Donec id elit non mi porta.</small>
          </a>
        </div>

      </aside>
    );
  }
}

Sidebar.propTypes = {

};

export default Sidebar;
