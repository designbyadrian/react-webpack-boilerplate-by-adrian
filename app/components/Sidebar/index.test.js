import React from 'react';
import { render } from 'react-testing-library';

import Sidebar from './index';

describe('components/Sidebar', () => {
  it('matches snapshot', () => {
    const { container } = render(<Sidebar />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
