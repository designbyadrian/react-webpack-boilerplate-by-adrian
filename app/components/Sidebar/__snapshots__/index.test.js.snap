// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Sidebar matches snapshot 1`] = `
<aside
  class="style-module-sidebar"
>
  <div
    class="list-group"
  >
    <a
      class="list-group-item list-group-item-action flex-column align-items-start"
      href="#"
    >
      <div
        class="d-flex w-100 justify-content-between"
      >
        <h5
          class="mb-1"
        >
          List group item heading
        </h5>
        <small>
          3 days ago
        </small>
      </div>
      <p
        class="mb-1"
      >
        Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
      </p>
      <small>
        Donec id elit non mi porta.
      </small>
    </a>
    <a
      class="list-group-item list-group-item-action flex-column align-items-start"
      href="#"
    >
      <div
        class="d-flex w-100 justify-content-between"
      >
        <h5
          class="mb-1"
        >
          List group item heading
        </h5>
        <small
          class="text-muted"
        >
          3 days ago
        </small>
      </div>
      <p
        class="mb-1"
      >
        Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
      </p>
      <small
        class="text-muted"
      >
        Donec id elit non mi porta.
      </small>
    </a>
    <a
      class="list-group-item list-group-item-action flex-column align-items-start"
      href="#"
    >
      <div
        class="d-flex w-100 justify-content-between"
      >
        <h5
          class="mb-1"
        >
          List group item heading
        </h5>
        <small
          class="text-muted"
        >
          3 days ago
        </small>
      </div>
      <p
        class="mb-1"
      >
        Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
      </p>
      <small
        class="text-muted"
      >
        Donec id elit non mi porta.
      </small>
    </a>
  </div>
</aside>
`;
